examples/get_target_descriptions.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+###############################################################################
+# SKA South Africa (http://ska.ac.za/)                                        #
+# Author: cam@ska.ac.za                                                       #
+# Copyright @ 2013 SKA SA. All rights reserved.                               #
+#                                                                             #
+# THIS SOFTWARE MAY NOT BE COPIED OR DISTRIBUTED IN ANY FORM WITHOUT THE      #
+# WRITTEN PERMISSION OF SKA SA.                                               #
+###############################################################################
+"""Simple example demonstrating getting pointing information by supplying a
+reference observer location and time.
+
+This example uses HTTP access to katportal, not websocket access.
+"""
+import logging
+import time
+
+import tornado.gen
+
+from katportalclient import KATPortalClient
+
+
+logger = logging.getLogger('katportalclient.example')
+logger.setLevel(logging.INFO)
+
+
+@tornado.gen.coroutine
+def main():
+    # Change URL to point to a valid portal node.  Subarray can be 1 to 4.
+    # Note: if on_update_callback is set to None, then we cannot use the
+    #       KATPortalClient.connect() method (i.e. no websocket access).
+    client = KATPortalClient('http://localhost/api/client/1',
+                             on_update_callback=None, logger=logger)
+
+    results = yield client.future_targets('20170123-0017')
+    print results
+
+    # Example output:
+    # [{u'target': u'Moon', u'slew_time': 53.6153013706, u'track_duration': 4.0}]
+
+    client.set_reference_observer_config(
+        longitude=10.0,
+        latitude=10.0,
+        altitude=10.0,
+        timestamp=time.time() + 3600)  # one hour in the future
+
```

Comment:
My idea was that MeerLICHT could use this to figure out their own pointing coordinates using the SB's future targets list and use their telescope as the reference observer. With that they can point to the same thing when tracking the target we track by using our catalogues + config_label.

The `future_targets` method gives a lot less detail. It's only the `future_targets_details` method that goes and works out pointing.

I might have over engineered it a bit, but I wanted to be quite flexible with this for them.